backend/data/src/main/kotlin/io/tolgee/model/contentDelivery/ContentDeliveryConfig.kt

```diff
@@ -129,4 +129,7 @@ class ContentDeliveryConfig(
 
   @ActivityLoggedProp
   override var fileStructureTemplate: String? = null
+
+  @ActivityLoggedProp
+  override var escapeHtml: Boolean? = false
```

Comment:
There is no reason to make it nullable. Make it false by default and add `@ColumnDefault` annotation so tests won't fail.